src/i18n/de/de.ts

```diff
@@ -0,0 +1,25 @@
+const translation = {
+  pages: {
```

Comment:
I wouldn't have this as a nested object, I'd just have the key be the default translation and the value be the translated text